src/renderer/pages/modelling-ui-poc/column-group/column-group.tsx

```diff
@@ -0,0 +1,171 @@
+import { useEffect } from 'react';
+import { WebGLRenderer, OrthographicCamera, Scene, Color, Group } from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+import invariant from 'tiny-invariant';
+import { InfiniteGrid, ModelGrid, Column } from 'renderer/three/components';
+import { CSS3DRenderer } from 'three/examples/jsm/renderers/CSS3DRenderer';
+import { COLORS } from 'renderer/constants';
+
+const ColumnSystem = () => {
+  useEffect(() => {
+    const webglContainerEl = document.querySelector('.webgl-wrapper');
+    invariant(webglContainerEl, 'could not find container tag');
+
+    const renderer = new WebGLRenderer({ alpha: true });
+    renderer.setSize(
+      webglContainerEl.clientWidth,
+      webglContainerEl.clientHeight
+    );
+
+    webglContainerEl.appendChild(renderer.domElement);
+
+    const cssContainerEl = document.querySelector('.css-wrapper');
+    invariant(cssContainerEl, 'could not find container tag');
+
+    const cssRenderer = new CSS3DRenderer();
+    cssRenderer.setSize(
+      cssContainerEl.clientWidth,
+      cssContainerEl.clientHeight
+    );
+
+    cssContainerEl.appendChild(cssRenderer.domElement);
+
+    const camera = new OrthographicCamera(
+      -webglContainerEl.clientWidth / 2,
+      webglContainerEl.clientWidth / 2,
+      webglContainerEl.clientHeight / 2,
+      -webglContainerEl.clientHeight / 2,
+      0.1,
+      1000
+    );
+
+    const controls = new OrbitControls(camera, renderer.domElement);
+    const cssControls = new OrbitControls(camera, cssRenderer.domElement);
+
+    controls.enableRotate = false;
+    cssControls.enableRotate = false;
+
+    camera.position.set(0, 0, 10);
+    controls.update();
+    cssControls.update();
+
+    const scene = new Scene();
+    scene.background = new Color(COLORS.GRAY_DARK);
```

Comment:
I'm seeing you changed the background and infinite grid colors. Any reason to do that or was it just to play around with it?

Reply:
just to play around, acttually I think we had a not valid color and I put a another one